assets/constants.ts

```diff
@@ -7057,6 +7057,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "issue-resp",
+  "SHORT_COMMIT_HASH": "24917f4a8"
```

Comment:
Revert these changes.

Reply:
I have revert the changes can you please now review pr ?